pkg/db/canary.go

```diff
@@ -18,7 +20,26 @@ import (
 func GetAllCanaries() ([]v1.Canary, error) {
 	var canaries []v1.Canary
 	var _canaries []pkg.Canary
-	if err := Gorm.Where("deleted_at is NULL").Find(&_canaries).Error; err != nil {
+	var rawCanaries interface{}
+	sql := fmt.Sprintf("SELECT json_agg(jsonb_set_lax(to_jsonb(canaries),'{checks}', %s)) :: jsonb as canaries from canaries where deleted_at is null", getChecksForCanaries())
```

Comment:
prefer using the variable `query` to avoid confusion with the `database/sql` package